native/packages/react-native-bpk-component-text-input/src/BpkTextInputStyles.js

```diff
@@ -0,0 +1,126 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { StyleSheet } from 'react-native';
+import {
+  colorBlue500,
+  colorGray100,
+  colorGray300,
+  colorGray500,
+  colorGray700,
+  colorRed500,
+
+  spacingSm,
+  borderSizeSm,
+
+  fontFamily,
+
+  textSmLineHeight,
+  textSmFontSize,
+  textSmFontWeight,
+  textBaseLineHeight,
+  textBaseFontSize,
+  textBaseFontWeight,
+} from 'bpk-tokens/tokens/base.react.native';
+
+const INPUT_RANGE = [0, 1];
+
+const styles = StyleSheet.create({
+  container: {
+    paddingTop: textSmLineHeight,
+  },
+  input: {
+    flex: 1,
+    paddingTop: spacingSm,
+    paddingRight: 0,
+    paddingBottom: spacingSm,
+    paddingLeft: 0,
+    minHeight: textBaseLineHeight + (spacingSm * 2),
```

Comment:
isn't `spacingSm * 2` `spacingMd`? we could use that

Reply:
Yep I believe it is, good catch

Reply:
Changed